### PR TITLE
ajout formulaire de base pour utilisateur inscription cours

### PR DESCRIPTION
**Pull request by ChristianOlmedo**

### CODE REVIEW

**Review by Loic-Code — DISMISSED** [review]

File: src/Controller/CourseFormController.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+
+namespace App\Controller;
+
+class CourseFormController extends AbstractController
+{
+    public function inscription()
```
Comment (LucyMerlier) [inline src/Controller/CourseFormController.php]: need return typehint

File: src/View/User/Includes/_inscription_form_course.html.twig [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                        <option>Non</option>
+                    </select>
+                </div>
+                <div class="mb-3 champ">
+                    <p> A quel cours souhaitez vous inscrire votre enfant ? :</p>
+                    <select name="course" class="form-label">
+                        <option></option>
+                        <option></option>
+                        <option></option>
+                        <option></option>
```
Comment (LucyMerlier) [inline src/View/User/Includes/_inscription_form_course.html.twig:33]: no placeholders while waiting for db values?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

File: src/View/User/Includes/_inscription_form_course.html.twig (hunk 2 above)
Comment (ChristianOlmedo) [reply src/View/User/Includes/_inscription_form_course.html.twig:33]: Exactly for now i haven't linked this page with data bases, i'm waiting for their validation

File: src/Controller/CourseFormController.php (hunk 1 above)
Comment (ChristianOlmedo) [reply src/Controller/CourseFormController.php]: True... done !

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**